projects/Heavenly Jungle NFT

```diff
@@ -0,0 +1,53 @@
+## Heavenly Jungle NFT
+
+## Category 
+Application
+
+## Project Description
+DAO NFT Platform allowing fractional ownership. Includes minting, auctions, royalties, licensing and the option to rage quit.
+
+## Use of IPFS, Filecoin and Libp2p
+Currently using Pinata's IPFS pinning service. Looking for guidance on this matter.
+
+## Project Status
+Early Development
+
+## Previews
+<!--Add some screenshots to give a preview of your product-->
```

Comment:
Please add screenshot

Reply:
![NFT View](https://ipfs.io/ipfs/QmTvZq1BDZ5MnyC9DnFPvr3uMp8cLRF7iR8gpMmgfU6kND)
![About Commons](https://ipfs.io/ipfs/QmYTHHUubnLZUuUs5NVWMbCrvQ6DN5Z5pihkXyRsJJmaKa)
![Administate the Commons](https://ipfs.io/ipfs/QmTGWaTVzzKmzxMrkp4XNt6xv8XYdaa6Engup12qprKik5)
![Commons Gallery](https://ipfs.io/ipfs/QmRHbJKQLGNtQVaW9F2uj9Dsnk2g78sEACK5BGRn5J1AyW)